### PR TITLE
Stop setting `defaultPolicies.remove=true` in cilium-app

### PR DESCRIPTION
**Pull request by kopiczko**
/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-cloud-director/values.schema.json
Comment (kopiczko) [inline helm/cluster-cloud-director/values.schema.json:1]: CI told me to run `schemalint normalize ./helm/cluster-cloud-director/values.schema.json -o ./helm/cluster-cloud-director/values.schema.json --force` so I did

**Comment by kopiczko** [comment]
In the test default-apps were not installed, but I checked the cluster and it looks like they were.

```
$ k get app -n org-t-031tghjbldq7ixqikv   t-e7ew2e1ao4g049mhem-default-apps    
NAME                                INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
t-e7ew2e1ao4g049mhem-default-apps   0.14.0              23h          23h             deployed
                                                                                             
```

**Comment by vxav** [comment]
This is slightly suspicious. Running the tests again, otherwise we should investigate in case the csi lost access to the vcd api for some ciliumy reason
```
      <*errors.errorString | 0xc000f4c090>: 
      pod pvc-test-pod in namespace test-storage is not running
      {
          s: "pod pvc-test-pod in namespace test-storage is not running",
      }
```

/run cluster-test-suites

**Review by vxav — APPROVED** [review]

**Comment by kopiczko** [comment]
E2E was green before main merge so proceeding.

**Pull request merged**